com.adsbynimbus.nimbus/Runtime/Scripts/Nimbus.Internal/NimbusAdUnit.cs

```diff
@@ -132,7 +132,7 @@ await Task.Run(async () => {
 					_adEvents.FireOnAdErrorEvent(this);
 					return;
 				}
-
+				Debug.Log("BID RESPONSE: " + response);
```

Comment:
Other places in code use `Debug.unityLogger` when accessing the logger, is there anything different that happens when using `Debug.Log` instead?

Separately it might help to add some kind of tag or something to indicate this came from the Nimbus SDK.